verilog/analysis/checkers/unpacked_dimensions_rule_test.cc

```diff
@@ -109,6 +109,13 @@ TEST(UnpackedDimensionsRuleTests, CheckRanges) {
       // inner range is packed dimensions, should not be detected as unpacked
       {"int some_array [bit [7:0]];"},
       {"int some_array [bit [0:7]];"},
+
+      // array of instances
+      {"module test (input logic [1:0] blah);",
+       "endmodule",
```

Comment:
This looks like a packed-dimensions usage in the ports, which is not checked by this rule.  It's ok to keep this as a negative test case, but I'd re-arrange the comment to reflect what covers `// array of instances`, below.

Reply:
You're right. I'll remove the packed-dimensions in ports just to keep thing clear.  Thanks for feedback.